packages/charts/examples/components/demo/advanced-usage/events/event-sampler/event-sampler.component.ts

```diff
@@ -125,6 +127,11 @@ export class EventSamplerComponent implements OnInit {
             id: CHART_VIEW_STATUS_EVENT,
             name: "CHART_VIEW_STATUS_EVENT",
         },
+        {
+            id: SERIES_STATE_CHANGE_EVENT,
+            name: "SERIES_STATE_CHANGE_EVENT",
+            interactionTypes: [InteractionType.MouseMove],
```

Comment:
I think the interactionTypes property can be omitted since it's not applicable in this case.